pkg/util/archive/tgz.go

```diff
@@ -62,6 +65,8 @@ func (t *Tgz) Unarchive(r io.Reader, handler FileHandler) error {
 			if err = handler(tf); err != nil {
 				return err
 			}
+		default:
+			return fmt.Errorf("Unknown header typeflag:  %v", header.Typeflag)
```

Comment:
There's a whole bunch of other valid (but skippable) typeflags - I don't think we need to error out if encountering them. Perhaps remove the `default` and just handle the types we care about?

Reply:
Removed the case for TypeDir as well since that was explicitly skipped previously and rebased :+1: